client/src/components/TreeDropdownField/TreeDropdownField.js

```diff
@@ -405,10 +423,15 @@ class TreeDropdownField extends Component {
    */
   renderMenu(renderMenuOptions) {
     // Build root node
-    const tree = this.getVisibleTree() || {};
+    const visibleTree = this.getVisibleTree();
+    const tree = Object.assign({}, visibleTree, {
+      // we only want to show options with a title
```

Comment:
An empty title is fine and that wouldn't get filtered out, is there a case when a File would have a title of `null` instead of an empty string?

P.S. Please comment on the correct line you're referring to, I couldn't see the context of your question